File: Results/DER/Translation/translation_output/w_test/codeqwen/codenet/java/python/p02111.py
Here is the Python equivalent of the Java code:

```python
n = int(input())
minute = 12 * 60 * n / 360
print((minute // 60) + " " + str(minute % 60))
```

In Python, the `//` operator is used for integer division, which is equivalent to the `/` operator in Java but returns an integer result. The `%` operator is used for the modulo operation, which is equivalent to the `%` operator in Java but returns the remainder of the division. The `str()` function is used to convert the remainder to a string so that it can be concatenated with the hour value.

File: Results/DER/Translation/translation_output/w_test/codeqwen/codenet/python/java/p02111.java
Here is the Java equivalent of the Python code you provided:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int input = scanner.nextInt();
        int result = input * 2;
        int minutes = result % 60;
        int hours = result / 60;
        System.out.println(hours + " " + minutes);
    }
}
```

This Java program reads an integer from the user, multiplies it by 2, and then calculates the number of hours and minutes. It then prints out the hours and minutes in the format "hours minutes".
